HLTriggerOffline/Muon/python/hltMuonValidator_cfi.py

```diff
@@ -6,10 +6,10 @@
 
     hltProcessName = cms.string("HLT"),
     hltPathsToCheck = cms.vstring(
-        "HLT_(L[12])?(Iso)?(Tk)?Mu[0-9]*(Open)?(_NoVertex)?(_eta2p1)?(_v[0-9]*)?$",
+        "HLT_(HighPt)?(L[12])?(Iso)?(Tk)?Mu[0-9]*(Open)?(_NoVertex)?(_eta2p1)?(_v[0-9]*)?$",
```

Comment:
[CMSHLT-2277](https://its.cern.ch/jira/browse/CMSHLT-2277) renamed `HLT_TkMu100` to `HLT_HighPtTkMu100`, and this change adds monitoring for the latter.